README.md

```diff
@@ -49,22 +73,22 @@ $ sudo bash build.bash platform dev-url branch url
 Testing is done continuously with Travis using the test framework [Bats](https://github.com/bats-core/bats-core) and the linter [Shellcheck](https://www.shellcheck.net/).  As the tests focus on installing software, a docker solution is used for easy build-up and teardown. To run the test suite execute the commands below or `"$ ./test.bash docker-full"`. Docker and Shellcheck need to be installed. For more details regarding the tests see [Test Architecture](https://github.com/openhab/openhabian/blob/master/CONTRIBUTING.md#test-architecture) in CONTRIBUTING.
 
 ```
-docker build --tag openhabian/openhabian-bats .
-docker run -it openhabian/openhabian-bats bash -c 'bats -r -f "unit-." .'
+docker build -t openhabian/openhabian-bats .
+docker run -i openhabian/openhabian-bats bash -c 'bats -r -f "unit-." .'
 
 docker run --name "install-test" --privileged -d openhabian/openhabian-bats
-docker exec -it install-test bash -c "./build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local"
-docker exec -it install-test bash -c 'bats -r -f "installation-." .'
-docker exec -it install-test bash -c 'bats -r -f "destructive-." .'
+docker exec -i install-test bash -c "./build.bash local-test && mv ~/.profile ~/.bash_profil && /etc/rc.local"                                                
```

Comment:
Get rid of the extraneous spaces, and shouldn't `~/.bash_profil` be `~/.bash_profile`

Reply:
what extraneous spaces?

Reply:
perhaps it was just a bug with github, it looked like there were a bunch of extra spaces on the end of line 80